syfertext/pipeline/sentencizer.py

```diff
@@ -0,0 +1,123 @@
+from syfertext.doc import Doc
+from syfertext.token import Token
+
+from syft.workers.base import BaseWorker
+import syft.serde.msgpack.serde as serde
+
+
+class Sentencizer:
```

Comment:
Please check `pipeline/simple_tagger.py` as an example of how classes should be registered in PySyft's serde after I have added support for pysyft version > 0.2.6. The class should inherit `AbstractSendable` and override `get_msgpack_code`.